packages/fxa-content-server/app/scripts/views/post_verify/account_recovery/save_recovery_key.js

```diff
@@ -52,6 +52,18 @@ class SaveRecoveryKey extends FormView {
     this.print(PrintTemplate({ recoveryKey: this.recoveryKey }));
   }
 
+  // This method typically targets a similar success banner
+  // whose styling and visibility depends on it being contained
+  // inside a modal. Since we don't want that here the more
+  // straightforward approach is to just override this method
+  // to target our other styled success banner.
+  _displaySuccess(msg) {
```

Comment:
I think we can pull this logic into https://github.com/mozilla/fxa/blob/8701348cdd79dbdc9879b2b4a55a23a135a32bc1/packages/fxa-content-server/app/scripts/views/mixins/save-options-mixin.js#L85 so that it lives in one place. Depending if modal class exists it would apply the correct styles.